### ISSUE TITLE
Antigen fails to run after upgrading to 1.2.2

### ISSUE THREAD

**Issue by john-kurkowski** [zsh-users/antigen#317]
I upgraded Antigen and opened a new shell. Antigen fails to start; it doesn't load my theme. I saw:

```zsh
sed: 1: " /\{$/,/^\}/!{s/^local  ...": bad flag in substitute command: '}'
```

Opening a new shell after that has no error, but still doesn't load my theme. I can reproduce the error on every new shell if I first run `antigen reset`.

* macOS 10.12.1
    * Workaround: I already had 1.2.1 installed, so I downgraded via `brew switch antigen 1.2.1` and I'm back in business.
* zsh 5.2 dev-1

**Comment by john-kurkowski** [comment]
That regex was updated between 1.2.1 and 1.2.2, in [this commit/file](https://github.com/zsh-users/antigen/commit/42b0097c1f1036e7bdf1944c0bc24397f94a6dba#diff-72faec5cf5ecca2e084494d130e482ce).

**Comment by desyncr** [comment]
Should be fixed in [v1.2.3](https://github.com/zsh-users/antigen/milestone/12)
